apps/spruce/src/pages/version/VersionTabs.tsx

```diff
@@ -20,20 +20,59 @@ const { parseQueryString } = queryString;
 interface Props {
   taskCount: number;
   isPatch: boolean;
-  // @ts-expect-error: FIXME. This comment was added by an automated script.
-  childPatches: VersionQuery["version"]["patch"]["childPatches"];
+  childPatches: NonNullable<VersionQuery["version"]["patch"]>["childPatches"];
```

Comment:
nit. You can extract this into its own type and reuse it below instead of repeating the code